bcda/service/service.go

```diff
@@ -673,3 +678,17 @@ var (
 	ErrJobNotCancelled   = goerrors.New("Job was not cancelled due to internal server error.")
 	ErrJobNotCancellable = goerrors.New("Job was not cancelled because it is not Pending or In Progress")
 )
+
+type CtxACOCfgType string
```

Comment:
Is there a better place for this type and the following?  Thinking maybe models?

Reply:
Yeah, I agree. I spent quite a bit of time debating this yesterday and messing around with placement and struggled to settle on where to put these. 

In my mind, this key,value would be set in the middleware after the auth data is added to the context, but perhaps the types could live in a more common package, such as models. 

Since GetACOConfigByID is a method of Service and our package structure might benefit from combining some packages and moving things around, I wanted to get y'alls thoughts on that before making changes. Maybe we could discuss more?

Reply:
Per our convo, we are going to keep this here. It really should live elsewhere (ideally middleware), but this is consistent with other k,v pairs that get added to the context in other parts of the project. We are going to discuss how we can do some minor restructuring of packages to make changes like this cleaner in the future.